src/libLookup/Lookup.cpp

```diff
@@ -132,21 +134,29 @@ void Lookup::GetInitialBlocksAndShardingStructure() {
   }
 }
 
+void Lookup::UpdateAllSeedsAndMultipliers() {
+  LOG_MARKER();
+
+  SetLookupNodes();
```

Comment:
lookup does not support scalable feature for reason line coinbase reward.
And we don't forsee any need to scale lookup since they are not involved in syncing or api service